system-addon/lib/SectionsManager.jsm

```diff
@@ -78,6 +81,7 @@ const SectionsManager = {
   initialized: false,
   sections: new Map(),
   init(prefs = {}) {
+    ABOUT_PREFERENCES = prefs["feeds.aboutpreferences"];
```

Comment:
This should be temporary only for experimentation. The idea is for implementing bug 1432672 to open about:preferences instead of sidebar, it would also look at this same feed pref.